tests/integration_tests.rs

```diff
@@ -2631,3 +2631,62 @@ fn highlighting_independant_from_map_syntax_case() {
         .stdout(expected)
         .stderr("");
 }
+
+// Tests that style components can be removed with `-component`.
+#[test]
+fn style_components_can_be_removed() {
+    bat()
+        .arg("--style=full,-grid")
+        .arg("--decorations=always")
+        .arg("--color=never")
+        .write_stdin("test")
+        .assert()
+        .success()
+        .stdout("       STDIN\n       Size: -\n   1   test\n")
+        .stderr("");
+}
```

Comment:
On the MSRV toolchain, `cargo test` is run without the `git` feature. Therefore, the style component `full` does not contain `changes`, so less space is allocated for the sidebar.
```suggestion
#[test]
fn style_components_can_be_removed() {
    bat()
        .arg({
            #[cfg(not(feature = "git"))]
            {
                "--style=full,-grid"
            }
            #[cfg(feature = "git")]
            {
                "--style=full,-changes,-grid"
            }
        })
        .arg("--decorations=always")
        .arg("--color=never")
        .write_stdin("test")
        .assert()
        .success()
        .stdout("     STDIN\n     Size: -\n   1 test\n")
        .stderr("");
}
```